### ISSUE TITLE
codes

### ISSUE THREAD

**Issue by ADITYA-THEE-CODER** [ADITYA-THEE-CODER/ML-#78]
import numpy as np
from sklearn.preprocessing import StandardScaler
from sklearn.neighbors import KNeighborsClassifier

# data
data = np.array([
    [20, 20000],
    [25, 30000],
    [30, 80000],
    [35, 120000]
])

labels = np.array([0, 0, 1, 1])
